test/scripts/transforms/test_iris_transform.py

```diff
@@ -15,7 +15,7 @@ def generate_input_path_for_run(run_number, cycle):
 
 # To change by automatic script
 input_runs = [105277]
-calibration_run_number = "105277"
```

Comment:
Changing this calibration_run_number from a string to a list is not consistent with other similar changes (see other comment)